docker/rstudio/Dockerfile

```diff
@@ -21,6 +21,11 @@ RUN R -e 'install.packages(c( \
     repos="http://cran.mtu.edu")' \
  && R -e 'devtools::install_github("DataBiosphere/Ronaldo")'
 
+# install python dev libs
```

Comment:
Could you put this in the above layer which starts with `RUN apt-get update`? In general fewer layers is better for docker images because it reduces image size, etc.

Reply:
💯